index.html

```diff
@@ -5,27 +5,29 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
-    <script src="main.jss" defer></script>
+    <script src="main.js" defer></script>
     <title>TodoList</title>
 </head>
 <body>
     <div id="flex-container">
```

Comment:
Don't use ```id``` for styling use ```class```, until it's unique in the HTML.

Reply:
Done sir